src/test/java/io/github/sashirestela/cleverclient/metadata/InterfaceMetadataStoreTest.java

```diff
@@ -135,23 +135,22 @@ void shouldSaveInterfaceMetadataWhenAccomplishValidtion() {
     void shouldThrownExceptionWhenTryingToGetNotPreviouslySavedInterface() {
         Exception exception = assertThrows(CleverClientException.class,
                 () -> store.get(ITest.SyncService.class));
-        assertTrue(exception.getMessage().equals(
-                "The interaface SyncService has not been saved yet."));
+      assertEquals("The interface SyncService has not been saved yet.", exception.getMessage());
     }
 
     @Test
     void shouldThrownExceptionWhenMethodHasNotHttpAnnotation() {
         Exception exception = assertThrows(CleverClientException.class,
                 () -> store.save(ITest.NotAnnotatedService.class));
-        assertTrue(exception.getMessage().equals(
-                "Missing HTTP anotation for the method unannotatedMethod."));
+      assertEquals("Missing HTTP annotation for the method unannotatedMethod.",
+          exception.getMessage());
     }
 
     @Test
     void shouldThrownExceptionWhenUrlPathParamAtMethodUnmatchesAnnotatedArguments() {
         Exception exception = assertThrows(CleverClientException.class,
                 () -> store.save(ITest.BadPathParamService.class));
-        assertTrue(exception.getMessage().equals(
-                "Path param demoId in the url cannot find an annotated argument in the method unmatchedPathParamMethod."));
+        assertEquals(exception.getMessage(),
```

Comment:
If you are using `assertEquals`, you need to take in account the order of the arguments: first is the expected value. Please, change the order.